synfig-core/src/synfig/filesystemnative.cpp

```diff
@@ -70,6 +70,12 @@ std::istream::pos_type FileSystemNative::ReadStream::seekpos(std::istream::pos_t
 	return ftell(file_);
 }
 
+int FileSystemNative::ReadStream::pbackfail(int ch)
+{
+	fseek(file_, -1, SEEK_CUR);
```

Comment:
Would be good to check the returned value of `fseek()`?
`pbackfail()` should return `EOF` in case of failure.

Reply:
Done